src/Page/CIFpage/index.js

```diff
@@ -0,0 +1,246 @@
+            </div>
+          </div>
+          <span className="text-gray">
+            We encourage women to apply. This is a spectacular opportunity for
+            freshers and recent graduates who are passionate to work in the
+            field of social work.
+          </span>
+        </div>
+      </div>
+      <div className="description text-center d-flex flex-column align-items-center justify-content-center ">
+        <h3 className="mb-3">Role Description</h3>
+        <hr className="heading-hr mb-3" />
+        <div className="container text-start mb-1 mt-3 w-50">
+          <div className="sub-section">
+            <span className="mb-3 d-block fw-bold">Teaching</span>
+            <ul>
+              <li>
+                Support young children by teaching programming or coding through
+                application of innovative teaching techniques
+              </li>
+              <li>
+                Help with curriculum design involving its structure, forms of
+                delivery, method of assessment and approach to teaching
+              </li>
+              <li>
+                Mentor the students in their study and help them with learning
+                problems
+              </li>
+              <li>Formulate methods for evaluating teaching outcomes</li>
+            </ul>
+          </div>
+          <div className="sub-section">
+            <span className="mb-3 fw-bold d-block">
+              Managing people and resources
+            </span>
+            <ul>
+              <li>Manage your teaching and administrative activities</li>
+              <li>
+                Prioritise the tasks as per the agreed objectives to ensure they
+                are completed on time
+              </li>
+              <li>
+                Appreciating the individual needs of students, their
+                circumstances and tailoring the support according to it
+              </li>
+            </ul>
+          </div>
+          <div className="sub-section">
+            <span className="mb-3 fw-bold d-block">
+              Academic administration
+            </span>
+            <ul>
+              <li>
+                Undertaking various administrative responsibilities as required,
+                including assessment of progress, student attendance, teaching
+                quality, student experience, etc
+              </li>
+              <li>
+                Participating in regular interactions with other CIF fellows and
+                be a part of the peer-to-peer learning process
+              </li>
+            </ul>
+          </div>
+          <div className="sub-section mb-0">
+            <span className="mb-3 fw-bold d-block">
+              Documentation and reporting
+            </span>
+            To document your journey as a fellow and prepare a report at the end
+            of the Fellowship
+          </div>
+        </div>
+      </div>
```

Comment:
This could be a nested unordered list with each `<span className="mb-3 fw-bold d-block">` being changed to a `<li>` with `<h4>` headings